static/js/util/util_test.js

```diff
@@ -1,4 +1,5 @@
 /* global SETTINGS: false */
+/* eslint-disable no-global-assign */
```

Comment:
You should be able to remove this since another PR today removed the global assign of `SETTINGS`